commands/csr-slash.js

```diff
@@ -0,0 +1,63 @@
+const { SlashCommandBuilder, EmbedBuilder } = require('discord.js');
+const getCSR = require('../utils/getCSR');
+const buildRankEmbeds = require('../utils/buildRankEmbeds');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('csr')
+		.setDescription("Get the CSR for a user by entering their gamertag.")
+		.addStringOption((option) =>
+		option
+		  .setName("gamertag")
+		  .setDescription("A valid Xbox Live gamertag")
+		  .setMaxLength(15)
+		  .setRequired(true)
+	  	),
+	async execute(interaction) {
+		try {
+			// console.log(`csr-slash.js - About to set const: gamertag`)
+			const gamertag = interaction.options.getString('gamertag');
+			console.log(`csr-slash.js - New value of const gamertag is:`)
+			console.log(gamertag)
```

Comment:
Same goes for `console.log` statements - mind removing these? If they're not necessary in prod they'll use up our log space.

Reply:
Sure thing, I'll get these unnecessary logs and clean up the code that is commented.